pkg/kapp/resources/identified_resources_pods.go

```diff
@@ -28,8 +29,15 @@ func (w UniquePodWatcher) Watch(podsToWatchCh chan corev1.Pod, cancelCh chan str
 	nonUniquePodsToWatchCh := make(chan corev1.Pod)
 
 	go func() {
+		namespace := ""
+		if len(w.fallbackAllowedNamespaces) > 0 {
```

Comment:
fallbackAllowedNamespaces is always set which means that we would always limit it to this namespace. we should use it when `errors.IsForbidden(err)` is true for the Watch call.

Reply:
Got it. Can we assume fallbackAllowedNamespaces[0] should be used for watch pods if fallbackAllowedNamespaces[0] is not "default"?

Reply:
@cppforlife I added a new commit to follow your advise.